src/devices/Max7219/samples/Max7219.sample.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+using System.Device.Spi.Drivers;
+using System.Threading;
+
+namespace Iot.Device.Samples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello Max7219!");
+
+            var connectionSettings = new SpiConnectionSettings(0, 0)
+            {
+                ClockFrequency = 10_000_000,
+                Mode = SpiMode.Mode0
+            };
+            var spi = new UnixSpiDevice(connectionSettings);
+            var devices = new Max7219(spi, cascadedDevices: 4);
+            using (devices)
+            {
+                Console.WriteLine("Display-Test");
+                devices.SetRegister(Max7219.Register.DISPLAYTEST, 1);
+                Thread.Sleep(1000);
+                Console.WriteLine("Init");
+                devices.Init();
+                foreach (Max7219.RotationType rotation in Enum.GetValues(typeof(Max7219.RotationType)))
```

Comment:
Can we add comments in the sample code in order to understand what are the different values of RotationType?